sources/academy/tutorials/node_js/block_requests_puppeteer.md

```diff
@@ -5,27 +5,29 @@ sidebar_position: 16.2
 slug: /node-js/block-requests-puppeteer
 ---
 
-_Unfortunately, in the recent version of Puppeteer, request interception disables native cache and slows down the actor significantly. Therefore it is not recommended to follow examples shown in this article. Instead, use_ [_blockRequests_](/sdk/js/docs/api/puppeteer#puppeteerblockrequestspage-options-promise) _utility function from_ [_Apify SDK_](/sdk/js/)_. It works through different paths and doesn't slow down your process._
+:::caution Improve Performance: Use `blockRequests`
+
+Unfortunately, in the recent version of Puppeteer, request interception disables the native cache and slows down the Actor significantly. Therefore, it's not recommended to follow the examples shown in this article. Instead, use [<code>blockRequests</code>](https://crawlee.dev/api/puppeteer-crawler/namespace/puppeteerUtils#BlockRequestsOptions) _utility function from_ [_Crawlee_](https://crawlee.dev). It works through different paths and doesn't slow down your process.
+
+:::
 
 When using Puppeteer, often a webpage will load many resources that are not actually necessary for your use case. For example page could be loading many tracking libraries, that are completely unnecessary for most crawlers, but will cause the page to use more traffic and load slower.
 
 For example for this web page: <https://edition.cnn.com/>
-If we run an [Actor](https://www.apify.com/jaroslavhejlek/measure-downloaded-bytes) that measures extracted downloaded data from each response until the page is fully loaded, we get these results:
+If we run an Actor that measures extracted downloaded data from each response until the page is fully loaded, we get these results:
 
 ![Actor loading](./images/actor-load.png)
 
 
-And [this is how the website it looks](https://api.apify.com/v2/key-value-stores/sE2s9WmvoWFZhTff7/records/debug-screen.png?disableRedirect=true).
-
-Now if we want to optimise this to keep the webpage looking the same, but ignore unnecessary requests, then after
+Now if we want to optimize this to keep the webpage looking the same, but ignore unnecessary requests, then after
 
-```js
+```js showLineNumbers
```

Comment:
side note - do you think we should enable line numbers globally? docusaurus does not allow it right now, but we could try this [css hack](https://gist.github.com/uditalias/3969a20ef759bf3a8757499429b30063), or have a custom code block component override to add it (that could probably also allow opting out of them)

Reply:
fyi this is how it looks with the css solution

<img width="817" alt="image" src="https://github.com/apify/apify-docs/assets/615580/4cba0bb4-23a4-4abc-a588-cb6c6fc2c22d">

Reply:
here is a PR doing that globally, i'd probably go with that approach and remove the explicit marks you added here so we don't end up with double numbering 

#873

Reply:
Generally speaking I'm a fan of having numbered lines. I think it just make the code samples look cleaner.
I think that is a good idea, I will remove those from here, and will search for others that may already be in the docs in separate PR and then we could go forward with merging the #873